rhizome/host/lib/vm_setup.rb

```diff
@@ -572,6 +572,8 @@ def download_boot_image(boot_image)
       # Images are presumed to be atomically renamed into the path,
       # i.e. no partial images will be passed to qemu-image.
       r "qemu-img convert -p -f qcow2 -O raw #{temp_path.shellescape} #{image_path.shellescape}"
+
+      File.delete(temp_path)
```

Comment:
should this have an `ensure` type thing going on, with a rescue for `Errno::ENOENT`?

Reply:
It looks like we have a helper as `rm_if_exists`. I used it now